tremor-cli/src/run.rs

```diff
@@ -229,7 +230,7 @@ impl Egress {
 }
 
 fn run_tremor_source(matches: &ArgMatches, src: String) -> Result<()> {
-    let raw = slurp_string(&src);
+    let raw = fs::read_to_string(&src);
```

Comment:
I would prefer keeping slurp_string as it gives better errors than `fs::read_to_string`. (i.e. tells a user what file couldn't be opend)

Reply:
or perhaps even better a `tremor-common` like a variant of read_to_string

https://github.com/tremor-rs/tremor-runtime/blob/main/tremor-common/src/file.rs